docs/advanced-features/middleware.md

```diff
@@ -31,7 +31,7 @@ To begin using Middleware, follow the steps below:
 npm install next@latest
 ```
 
-2. Create a `middleware.ts` (or `.js`) file at the same level as your `pages` directory
+2. Create a `middleware.ts` (or `.js`) file at the root of your Next.js project
```

Comment:
Hi, this was more accurate as previously described since if you have `src/pages` the middleware should be placed at `src/middleware.ts`.

Reply:
Cool, I was trying to add middleware in the pages directory which was present in the root `/pages` instead of `src/pages`. That's why I thought of updating the README :)